USE_CASES.md

```diff
@@ -98,4 +98,18 @@ public class Example {
     }
   }
 }
-```
\ No newline at end of file
+```
+
+<a name="domain_whitelabel"></a>
+# How to Setup a Domain Whitelabel
+
+You can find documentation for how to setup a domain whitelabel via the UI [here](https://sendgrid.com/docs/Classroom/Basics/Whitelabel/setup_domain_whitelabel.html) and via API [here](https://github.com/sendgrid/sendgrid-csharp/blob/master/USAGE.md#whitelabel).
```

Comment:
The link should be https://github.com/sendgrid/sendgrid-java/blob/master/USAGE.md#whitelabel